Xamarin.Forms.Core/Application.cs

```diff
@@ -49,6 +57,8 @@ public IAppLinks AppLinks
 			}
 		}
 
+		public Menu MainMenu => _menus;
```

Comment:
Maybe this should be an attached property as it wont really be serviced on all platforms